public/data/damoski2.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "damoski2",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/damoski2.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/damoski2",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AkogunOyindamo3",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Linkedin",
```

I recommend capitalizing the `i`. What do you think?
```suggestion
      "name": "Follow me on LinkedIn",
```

---

Yes that would be better😄